pkg/apis/extensions/validation/validation.go

```diff
@@ -699,7 +699,6 @@ func ValidateNetworkPolicySpec(spec *extensions.NetworkPolicySpec, fldPath *fiel
 		// TODO: Update From to be a pointer to slice as soon as auto-generation supports it.
 		for _, f := range i.From {
 			numFroms := 0
-			allErrs := field.ErrorList{}
```

Comment:
This fixes a bug where errors during validation weren't being reported.

If we don't think that the rest of this PR can get in to v1.3, then I think at least the diffs in this file should.